tests/plenary/org/autocompletion_spec.lua

```diff
@@ -339,6 +339,39 @@ describe('Autocompletion', function()
           { menu = '[Org]', word = 'Title without anchor' },
         }, result)
       end)
+
+      it('should work on relative paths', function()
+        local files = helpers.create_agenda_files({
+          'a.org',
+          'b/c.org',
+        }, {
+          '[[./  ',
+          '[[../  ',
```

Comment:
I don't want to be a nitpick, but I would prefer to clean each test a bit up to be specific for the case. Makes it easier for future contributions to not be distracted by code, which is actually irrelevant for the test case.

Reply:
I agree with you. 

The case here is mainly about paths, so the content does not really matter. So do you suggest me to add some extra info to the files?

Reply:
Actually I had a misconception, how the `create_agenda_files` function works. The position in the file table corresponds with the position in the content table.

So I think, this cannot be improved further with reasonable amount of effort.

Reply:
Fine, so we just keep it?